recording-oracle/src/modules/liquidity-score/liquidity-score.service.ts

```diff
@@ -147,11 +150,16 @@ export class LiquidityScoreService {
             );
 
         const scoreToUpload =
-          liquidityScore &&
-          Number.isFinite(liquidityScore) &&
-          liquidityScore > 0
-            ? Math.round(liquidityScore)
-            : 0;
+          Number.isFinite(rawScore) && rawScore > 0 ? rawScore : 0;
+        const fingerprint = `${scoreToUpload}-${lastTradeId ?? 'NA'}`;
```

Comment:
we might have a situatio where 2 users provide 0 liquidity, so the lastTradeId for both will be null, so the fingerprint will 0-NA, but it doesn't mean they are duplicated